README.md

```diff
@@ -21,6 +21,21 @@ You can install Sequeler by compiling from the source, here's the list of depend
  - `libgda-5.0`
  - `gtksourceview-3.0`
 
+**Arch Based**
+ - `vala`
+ - `granite-git`
+ - `libgee`
+ - `libxml2`
+ - `libgda`
+ - `gtksourceview3`
+ - `libgda-mysql`
+ - `libgda-postgres`
+
+Installation using AUR and Yaourt:
+```
+yaourt -S vala granite-git libgee libxml2 libgda gtksourceview3 libgda-mysql libgda-postgres
```

Comment:
Is there actually any AUR packages needed that are in this list? Otherwise it's not good advice to point people to the not-default packagemanager that can also be considered bad practice.

Reply:
Most of the packages come from the AUR.

Reply:
I would actually be surprised if that was the case. `granite-git` I can see come from there. But the rest shouldn't need to.

But I'm not an arch-user (anymore). I've just gone through creating a package for NixOS :smile:

I just find AUR to be very dodgy when it comes to more or less everything.